### ISSUE TITLE
[7.1.0.91] Fluid network crash - server starts, runs fine for 5min, then crashes with this

### ISSUE THREAD

**Issue by Komoden** [mekanism/Mekanism#1759]
---- Minecraft Crash Report ----
// I blame Dinnerbone.

Time: 28-08-14 19:22
Description: Exception in server tick loop

java.lang.NullPointerException: Exception in server tick loop
    at mekanism.common.FluidNetwork.onUpdate(FluidNetwork.java:257)
    at mekanism.api.transmitters.DynamicNetwork.tick(DynamicNetwork.java:267)
    at mekanism.api.transmitters.TransmitterNetworkRegistry.tickEnd(TransmitterNetworkRegistry.java:85)
    at mekanism.api.transmitters.TransmitterNetworkRegistry.onTick(TransmitterNetworkRegistry.java:73)
    at cpw.mods.fml.common.eventhandler.ASMEventHandler_263_TransmitterNetworkRegistry_onTick_ServerTickEvent.invoke(.dynamic)
    at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:51)
    at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:122)
    at cpw.mods.fml.common.FMLCommonHandler.onPostServerTick(FMLCommonHandler.java:247)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:590)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- System Details --
Details:
    Minecraft Version: 1.7.10
    Operating System: Windows 7 (amd64) version 6.1
    Java Version: 1.7.0_55, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 278458536 bytes (265 MB) / 1287806976 bytes (1228 MB) up to 5732302848 bytes (5466 MB)
    JVM Flags: 7 total; -Xms512M -Xmx5500M -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -XX:+UseParNewGC -XX:+CMSIncrementalPacing -XX:ParallelGCThreads=4
    AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    IntCache: cache: 0, tcache: 0, allocated: 15, tallocated: 96
    FML: MCP v9.05 FML v7.10.25.1208 Minecraft Forge 10.13.0.1208 91 mods loaded, 91 mods active
    mcp{9.05} [Minecraft Coder Pack](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    FML{7.10.25.1208} [Forge Mod Loader](server.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Forge{10.13.0.1208} [Minecraft Forge](server.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    CodeChickenCore{1.0.3.23} [CodeChicken Core](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    <CoFH ASM>{000} [CoFH ASM Data Initialization](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MobiusCore{1.2.2} [MobiusCore](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    NotEnoughItems{1.0.3.51} [Not Enough Items](NotEnoughItems-1.7.10-1.0.3.51-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    TConstruct-Preloader{0.1.1} [Tinkers Corestruct](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    bdlib{1.4.3.14} [BD Lib](bdlib-mc1710-1.4.3.14.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BiblioCraft{1.7.5} [BiblioCraft](BiblioCraft[v1.7.5][MC1.7.10].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Mantle{1.7.10-165.4bc3343} [Mantle](Mantle_mc1.7.10_0.3.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Natura{2.2.0} [Natura](natura-1.7.10-2.2.0-b1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BiomesOPlenty{2.0.2} [Biomes O' Plenty](BiomesOPlenty-1.7.2-2.0.2.932-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BiblioWoodsBoP{1.7} [BiblioWoods Biomes O'Plenty Edition](BiblioWoods[BiomesOPlenty][v1.7].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    IC2{2.2.619-experimental} [IndustrialCraft 2](industrialcraft-2-2.2.619-experimental.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BuildCraft|Core{6.0.17} [BuildCraft](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Forestry{3.0.0.90} [Forestry for Minecraft](forestry_1.7.10-3.0.0.90.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BiblioWoodsForestry{1.5} [BiblioWoods Forestry Edition](BiblioWoods[Forestry][v1.5].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BiblioWoodsNatura{1.3} [BiblioWoods Natura Edition](BiblioWoods[Natura][v1.3].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    CoFHCore{1.7.10R3.0.0B5} [CoFH Core](CoFHCore-[1.7.10]3.0.0B5-30.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ThermalFoundation{1.7.10R1.0.0B3} [Thermal Foundation](ThermalFoundation-[1.7.10]1.0.0B3-8.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ThermalExpansion{1.7.10R4.0.0B5} [Thermal Expansion](ThermalExpansion-[1.7.10]4.0.0B5-13.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BigReactors{0.4.0rc5} [Big Reactors](BigReactors-0.4.0rc5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BinnieCore{2.0-dev3} [Binnie Core](binnie-mods-2.0-dev3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Botany{2.0-dev3} [Botany](binnie-mods-2.0-dev3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ExtraBees{2.0-dev3} [Extra Bees](binnie-mods-2.0-dev3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ExtraTrees{2.0-dev3} [Extra Trees](binnie-mods-2.0-dev3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Genetics{2.0-dev3} [Genetics](binnie-mods-2.0-dev3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    AWWayofTime{v1.0.1g} [Blood Magic: Alchemical Wizardry](Blood Magic-v1.1.0%281.7.10%29.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BuildCraft|Builders{6.0.17} [BC Builders](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BuildCraft|Energy{6.0.17} [BC Energy](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BuildCraft|Factory{6.0.17} [BC Factory](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BuildCraft|Transport{6.0.17} [BC Transport](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    BuildCraft|Silicon{6.0.17} [BC Silicon](buildcraft-6.0.17.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    CarpentersBlocks{3.2.6} [Carpenter's Blocks](Carpenter's Blocks v3.2.6 - MC 1.7.10.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ComputerCraft{1.64pr4} [ComputerCraft](ComputerCraft1.64pr4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Dynmap{2.0.0-alpha-2-17} [Dynmap](Dynmap-2.0.0-alpha-2-forge-10.13.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded{1.7.10R2.8.0RC2} [MineFactory Reloaded](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    gendustry{1.3.2.18} [GenDustry](gendustry-mc1710-1.3.2.18.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    IC2NuclearControl{1.7.10.04} [Nuclear Control](ic2nc-1.7.10-1.7.10.04.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    IronChest{6.0.49.733} [Iron Chest](ironchest-1.7.10-6.0.49.733-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    JABBA{1.1.4} [JABBA](Jabba-1.1.4_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MetallurgyCore{4.0.3} [Metallurgy Core](MetallurgyCore-1.7.10-4.0.3.11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Baubles{1.0.1.5} [Baubles](Baubles-1.7.10-1.0.1.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Thaumcraft{4.2.0.0} [Thaumcraft](Thaumcraft-1.7.10-4.2.0.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Railcraft{9.2.2.4} [Railcraft](Railcraft_1.7.10-9.2.2.4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    UniversalElectricity{@MAJOR@.@MINOR@.@REVIS@} [Universal Electricity](universal-electricity-4.0.0.50-core.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ForgeMultipart{1.1.0.297} [Forge Multipart](ForgeMultipart-1.7.10-1.1.0.297-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Mekanism{7.1.0} [Mekanism](Mekanism-1.7.10-7.1.0.91.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MekanismGenerators{7.1.0} [MekanismGenerators](MekanismGenerators-1.7.10-7.1.0.91.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    TConstruct{1.7.10-1.6.0.build591} [Tinkers' Construct](TConstruct_mc1.7.10_1.6.0d39.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Metallurgy{4.0.4} [Metallurgy 4](Metallurgy-1.7.10-4.0.4.43.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ResonantEngine{@MAJOR@.@MINOR@.@REVIS@} [Resonant Engine](resonant-engine-2.0.0.82-core.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MFFS{@MAJOR@.@MINOR@.@REVIS@} [Modular Force Field System](mffs-4.1.0.23-core.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatAppliedEnergistics{1.7.10R2.8.0RC2} [MFR Compat: Applied Energistics](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatAtum{1.7.10R2.8.0RC2} [MFR Compat: Atum](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatBackTools{1.7.10R2.8.0RC2} [MFR Compat: BackTools](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatBuildCraft{1.7.10R2.8.0RC2} [MFR Compat: BuildCraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatChococraft{1.7.10R2.8.0RC2} [MFR Compat: Chococraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatExtraBiomes{1.7.10R2.8.0RC2} [MFR Compat: ExtraBiomes](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatForestry{1.7.10R2.8.0RC2} [MFR Compat: Forestry](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatForgeMicroblock{1.7.10R2.8.0RC2} [MFR Compat: ForgeMicroblock](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatIC2{1.7.10R2.8.0RC2} [MFR Compat: IC2](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatMystcraft{1.7.10R2.8.0RC2} [MFR Compat: Mystcraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatPams{1.7.10R2.8.0RC2} [MFR Compat: Pam's Mods](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatProjRed{1.7.10R2.8.0RC2} [MFR Compat ProjectRed](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatRailcraft{1.7.10R2.8.0RC2} [MFR Compat: Railcraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatRP2{1.7.10R2.8.0RC2} [MFR Compat: RP2](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatSufficientBiomes{1.7.10R2.8.0RC2} [MFR Compat: Sufficient Biomes](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatThaumcraft{1.7.10R2.8.0RC2} [MFR Compat: Thaumcraft](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatThermalExpansion{1.7.10R2.8.0RC2} [MFR Compat: Thermal Expansion](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatTwilightForest{1.7.10R2.8.0RC2} [MFR Compat: TwilightForest](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MineFactoryReloaded|CompatVanilla{1.7.10R2.8.0RC2} [MFR Compat: Vanilla](MineFactoryReloaded-[1.7.10]2.8.0RC2-557.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Morpheus{1.7.10-1.5.18} [Morpheus](Morpheus-1.7.10-1.5.18%28SERVER ONLY%29.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    NEIAddons{1.12.2.9} [NEI Addons](neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    NEIAddons|Botany{1.12.2.9} [NEI Addons: Botany](neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    NEIAddons|Forestry{1.12.2.9} [NEI Addons: Forestry](neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    NEIAddons|CraftingTables{1.12.2.9} [NEI Addons: Crafting Tables](neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    NEIAddons|ExNihilo{1.12.2.9} [NEI Addons: Ex Nihilo](neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    NetherOres{1.7.10R2.3.0RC3} [Nether Ores](NetherOres-[1.7.10]2.3.0RC3-84.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ObsidiPlates{3.0.0.18} [ObsidiPlates](ObsidiPlates-1.7.10-universal-3.0.0.18.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    MapWriter{2.1.1} [MapWriter](Opis-1.2.2a_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Opis{1.2.2a} [Opis](Opis-1.2.2a_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    harvestcraft{1.7.2d} [Pam's HarvestCraft](Pam's HarvestCraft 1.7.10a.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ServerTools{2.1.0.56} [ServerTools](ServerTools-CORE-1.7.10-2.1.0.56.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ServerTools|Backup{2.1.0.11} [ServerTools Backup](ServerTools-BACKUP-1.7.10-2.1.0.11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    TMechworks{1.7.10-67.34d1d9b} [Tinkers' Mechworks](TMechworks_mc1.7.10_0.2.8.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    witchery{0.20.5} [Witchery](witchery-1.7.10-0.20.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    McMultipart{1.1.0.297} [Minecraft Multipart Plugin](ForgeMultipart-1.7.10-1.1.0.297-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    aobd{2.2.3} [Another One Bites The Dust](AOBD-2.2.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    ForgeMicroblock{1.1.0.297} [Forge Microblocks](ForgeMultipart-1.7.10-1.1.0.297-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
    Mantle Environment: Environment healthy.
    TConstruct Environment: Environment healthy.
    Profiler Position: N/A (disabled)
    Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    Player Count: 1 / 1337; [EntityPlayerMP['Komoden'/332, l='ibkicraft1710', x=-1073,54, y=63,00, z=1633,68]]
    Is Modded: Definitely; Server brand changed to 'fml,forge'
    Type: Dedicated Server (map_server.txt)

**Comment by TheTux94** [comment]
Hello,
I got this error today after connecting an electric pump to a power source. It only happens in multiplayer, water pumps works fine on singleplayer worlds.

java.lang.NoSuchMethodError: net.minecraftforge.fluids.FluidStack.getFluidID()I
	at mekanism.common.FluidNetwork.tickEmit(FluidNetwork.java:168)
	at mekanism.common.FluidNetwork.onUpdate(FluidNetwork.java:248)
	at mekanism.api.transmitters.DynamicNetwork.tick(DynamicNetwork.java:321)
	at mekanism.api.transmitters.TransmitterNetworkRegistry.tickEnd(TransmitterNetworkRegistry.java:105)
	at mekanism.api.transmitters.TransmitterNetworkRegistry.onTick(TransmitterNetworkRegistry.java:91)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_184_TransmitterNetworkRegistry_onTick_ServerTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:138)
	at cpw.mods.fml.common.FMLCommonHandler.onPostServerTick(FMLCommonHandler.java:247)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:590)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

To fix this I had to user MCEdit to delete the water pump and the ticks related to that block.

The server starts fine but it crashes as soon as any player logs in.

**Comment by FluffyCloud** [comment]
Okay, Blood Altar 1 -> blood Tank -> Blood Altar 2.  Blood tank already had LP in it.  Blood altar 1 is successfully extracting and had filled up 3 tanks.  I moved 1 tank, and ran line from it to blood altar 2, but blood altar 2 wasn't upgraded much, so it didn't take much out of the tank.  Then i ran more line from blood altar 1 to the tank.  as soon as I placed the last piece of cable connecting the altar to the tank, we crashed.  Running regrowth.  Log provided.

https://gist.github.com/FluffyCloud/c06763ecd0bdb7e70f1f#file-crash-2015-11-15_12-13-17-server-txt
